shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -810,23 +810,6 @@ - (void)testInputActionContinueAction {
   OCMVerify([mockBinaryMessenger sendOnChannel:@"flutter/textinput" message:encodedMethodCall]);
 }
 
-- (void)testDisablingAutocorrectDisablesSpellChecking {
-  FlutterTextInputView* inputView = [[FlutterTextInputView alloc] initWithOwner:textInputPlugin];
-
-  // Disable the interactive selection.
-  NSDictionary* config = self.mutableTemplateCopy;
-  [inputView configureWithDictionary:config];
-
-  XCTAssertEqual(inputView.autocorrectionType, UITextAutocorrectionTypeDefault);
-  XCTAssertEqual(inputView.spellCheckingType, UITextSpellCheckingTypeDefault);
-
-  [config setValue:@(NO) forKey:@"autocorrect"];
-  [inputView configureWithDictionary:config];
-
-  XCTAssertEqual(inputView.autocorrectionType, UITextAutocorrectionTypeNo);
-  XCTAssertEqual(inputView.spellCheckingType, UITextSpellCheckingTypeNo);
-}
-
```

Comment:
Please don't delete tests for existing behaviour; they were added for a reason. 

You'll need to update the existing test to reflect the new expected behaviour and add additional testing to cover all four combinations of autocorrect/enableSuggestions.